### PR TITLE
[RPA-1260.3] Show only sharable users' emails when editing collection

### PR DESCRIPTION
**Pull request by star-in-the-jar**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

---

I agree to the terms of the RunBotics Contributor License Agreement.

### CODE REVIEW

**Review by star-in-the-jar — COMMENTED** [review]
Self cr

File: runbotics-orchestrator/src/main/java/com/runbotics/service/UserService.java [hunk 1]
```diff
@@ -374,6 +374,12 @@ public Page<AdminUserDTO> getAllManagedUsers(Pageable pageable) {
         return userRepository.findAll(pageable).map(AdminUserDTO::new);
     }
 
+    @Transactional(readOnly = true)
+    public List<User> getNonAdminUsers() {
+        List<User> activeUsersWithIds = userRepository.findAllActivatedNonAdmins();
+        return activeUsersWithIds.stream().filter(user -> !hasAdminRole(user)).collect(Collectors.toList());
```
Comment (marcin-depcik) [inline runbotics-orchestrator/src/main/java/com/runbotics/service/UserService.java]: Do we need to filter, even if you query for non-administrator users?

**Review by marcin-depcik — APPROVED** [review]

File: runbotics-orchestrator/src/main/java/com/runbotics/service/UserService.java (hunk 1 above)
Comment (preludium) [reply runbotics-orchestrator/src/main/java/com/runbotics/service/UserService.java]: if you need to filter your query result it means its a bad query

Comment (star-in-the-jar) [reply runbotics-orchestrator/src/main/java/com/runbotics/service/UserService.java]: yeah it's redundant

**Pull request merged**